packages/server/src/app.js

```diff
@@ -1,13 +1,11 @@
 import express from 'express';
-import cors from 'cors';
-import bodyParser from 'body-parser';
 import path from 'path';
 
 import { isApiExternal } from './net';
 import modules from './modules';
 import websiteMiddleware from './middleware/website';
 import graphiqlMiddleware from './middleware/graphiql';
-import graphqlMiddleware from './middleware/graphql';
+import getGraphqlServer from './middleware/graphql';
```

Comment:
It is no longer a middleware, so it shouldn't be located in `./middleware` folder